packages/gestalt/src/InternalLink.js

```diff
@@ -46,6 +46,9 @@ type Props = {|
   wrappedComponent: 'button' | 'iconButton' | 'tapArea',
 |};
 
+/**
+ * https://gestalt.pinterest.systems/Link
```

Comment:
This isn't quite right, as this component is used in number of components aside from Link

Reply:
@dangerismycat thanks for spotting - updated